### PR TITLE
Added few usefull comments in 'internal/*'

### PR DESCRIPTION
**Pull request by proggga**

### CODE REVIEW

File: internal/backup.go [hunk 1]
```diff
@@ -1,5 +1,7 @@
 package internal
 
+// 'Sentinel' mean a 'file' with meta/external-info about backup (in some cases)
```
Comment (usernamedt) [inline internal/backup.go:3]: Move this comment to `backup_sentinel_dto.go`?

Comment (usernamedt) [inline internal/backup.go:3]: Move this to `backup_sentinel_dto.go`?

Comment (usernamedt) [reply internal/backup.go:3]: We can add something more detailed:

WAL-G stores information about single backup in the following files:
1. Sentinel file - contains useful metadata, such as backup start time, backup size, etc.
2. (only in Postgres) Metadata file  - Postgres sentinel files can be quite large (> 1GB), so the metadata file is useful for the quick fetch of backup-related information.

Comment (proggga) [reply internal/backup.go]: added this description to backup.go, fixed

**Review by usernamedt — APPROVED** [review]

**Pull request merged**